faker_edu/tests/test_faker_highered.py

```diff
@@ -65,6 +65,17 @@ def test_faculty_title(self):
         self.assertIn(parts[1], faker_edu.DEPARTMENTS,
                       'Position segment not from department list.')
 
+    def test_sports(self):
+        """Test that generated sport is from the list."""
+        for _ in range(10):
+            sport = self.fake.sport()
+            parts = sport.split()
+            self.assertTrue(len(parts) == 2, 'Missing a word')
```

Comment:
By setting this to require two words it fails when it encounters a two word sport.  This should probably be `>=` 2.